### PR TITLE
LPS-165494 Move redirection configuration to redirect portlet

### PR DESCRIPTION
**Pull request by adolfopa**
# What is this about?

This PR moves the redirection pattern configuration from Site Settings into the Redirect portlet.

image

# How do I test this?

Simply go to the Redirections app; the usage hasn't changed with respect to the original.

### CODE REVIEW

File: modules/apps/redirect/redirect-web/src/main/java/com/liferay/redirect/web/internal/display/context/RedirectDisplayContext.java [hunk 1]
```diff
@@ -45,11 +45,20 @@ public RedirectDisplayContext(
 	public List<NavigationItem> getNavigationItems() {
 		return NavigationItemListBuilder.add(
 			navigationItem -> {
-				navigationItem.setActive(!isShowRedirectNotFoundEntries());
+				navigationItem.setActive(isShowRedirectEntries());
 				navigationItem.setHref(_renderResponse.createRenderURL());
 				navigationItem.setLabel(
 					LanguageUtil.get(_httpServletRequest, "redirects"));
```
Comment (AliciaGarciaGarcia) [inline modules/apps/redirect/redirect-web/src/main/java/com/liferay/redirect/web/internal/display/context/RedirectDisplayContext.java:51]: - [ ] The current redirections in the redirections app should be available in an "Alias" tab.

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]
maybe we should trim the pattern because if we copy-paste some with a blank space it save nothing.

**Comment by ambrinchaudhary** [comment]
Added classes to the form that add margins and make it responsive. 

Screenshot 2022-10-18 at 18 49 26

Let me know if I can help with anything  else.

**Review by ambrinchaudhary — APPROVED** [review]

File: modules/apps/redirect/redirect-web/src/main/java/com/liferay/redirect/web/internal/display/context/RedirectDisplayContext.java (hunk 1 above)
Comment (adolfopa) [reply modules/apps/redirect/redirect-web/src/main/java/com/liferay/redirect/web/internal/display/context/RedirectDisplayContext.java]: Done. See latest changes.

**Comment by adolfopa** [comment]
ci:test:sf

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**